src/Servers/Kestrel/Kestrel/test/KestrelConfigurationLoaderTests.cs

```diff
@@ -456,6 +457,126 @@ private void EndpointConfigSectionCanSetProtocols(string input, HttpProtocols ex
             Assert.True(ran3);
         }
 
+        [Fact]
+        public void EndpointConfigureSection_CanSetSslProtocol()
```

Comment:
These tests are failing on the CI because they need a certificate to be set.
https://dev.azure.com/dnceng/public/_build/results?buildId=690261&view=ms.vss-test-web.build-test-results-tab&runId=21427284&resultId=102722&paneView=debug

See https://github.com/dotnet/aspnetcore/blob/854555292407266e94396a90e9cfad336374f7c7/src/Servers/Kestrel/Kestrel/test/KestrelConfigurationLoaderTests.cs#L141

Reply:
Thank you, it would've taken me much longer to figure this out.